packages/material-ui-codemod/src/v5.0.0/hidden-down-props.test/expected.js

```diff
@@ -0,0 +1,7 @@
+<div>
+  <Hidden smDown />
+  <Hidden mdDown />
+  <Hidden lgDown />
+  <Hidden xlDown />
+  <Hidden xlDown />
+</div>
```

Comment:
Hidden is gone, so we might not need this codemod.

Reply:
👍🏼 Reverted